src/main/java/com/coveros/selenified/Browser.java

```diff
@@ -30,61 +37,85 @@
  *
  * @author Max Saperstone
  * @version 3.0.4
- * @lastupdate 5/16/2018
+ * @lastupdate 1/12/2019
  */
 public class Browser {
+
+    private static final Logger log = Logger.getLogger(Browser.class);
+
     public enum BrowserName {
         NONE, HTMLUNIT, FIREFOX, CHROME, INTERNETEXPLORER, EDGE, ANDROID, IPAD, IPHONE, OPERA, SAFARI, PHANTOMJS
     }
 
+    public static final String SCREENSIZE = "screensize";
+    public static final String BROWSER = "browser";
+    public static final String NAME = "name";
+    public static final String VERSION = "version";
+    public static final String PLATFORM = "platform";
+
+    private String browserInput;
     private BrowserName name;
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename field "name" to prevent any misunderstanding/clash with field "NAME" defined on line 52. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1845)